test/helpers.bash

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+set -u
+
+# Work around skip() in old BATS versions not handling substitutions properly
+# wrt. 'set -u'. (fixed in commit 9b2b659e5ba769752c0d00dadb3ad9b58f59cbaa).
+# This can be removed once all CI nodes have been setup with new enough BATS.
```

Comment:
would be nice to specify the version here (is it v1.3.0?)

Reply:
Yes, AFAICT the earliest tagged version with that fix/commit is `v1.3.0`. `git` describes it as `v1.2.1-17-g9b2b659` and there are no other tagged versions between `v1.2.1` and `v1.3.0`.

@kolyshkin, I can update that comment with the version info. That said, do we want to merge such a commit ? Or would it be better to instead change how tests are run so that we check BATS version, and clone/install the correct version if we find a too old (or just a different) one ?